scanpipe/models.py

```diff
@@ -3777,6 +3780,99 @@ def as_spdx(self):
         )
 
 
+class PackageScore(UUIDPKModel, PackageScoreMixin):
+
+    def __str__(self):
+        return self.score or str(self.uuid)
+
+    discovered_package = models.ForeignKey(
+        DiscoveredPackage,
+        related_name="discovered_packages_score",
+        help_text=_("The package for which the score is given"),
+        on_delete=models.CASCADE,
+        editable=False,
+        blank=True,
+        null=True,
+    )
+
+    @classmethod
+    @transaction.atomic()
```

Comment:
Not sure why this is required here? We are not doing multiple database updates that could crash and needs to be handled?

Reply:
Since we are updating the checks and score table at one go for that reason, I have kept it aromic

Reply:
@404-geek Could you provide an example that shows why atomic() is useful here?

Reply:
@404-geek You haven't address the question above yet ;)